docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -89,6 +89,7 @@ A list of the available extension methods is below:
 | `AddSelfActivatingSchema<>` | Registers the specified schema which will create instances of unregistered graph types during initialization | |
 | `AddSerializer<>`       | Registers the specified serializer | |
 | `AddSystemTextJson`     | Registers the serializer that uses System.Text.Json as its underlying JSON serialization engine | GraphQL.SystemTextJson |
+| `AddUnhandledExceptionHandler` | Configures the unhandler exception handler | |
```

Comment:
So far we use Add or Use consistently depending on if it's configuring execution options (order doesn't usually matter) vs wrapping the execution (order matters).

In this case the last call to `AddUnhandledExceptionHandler` is going to be the only effective one, just as the last call to `AddSerializer` is the only one effective.

I also called it `AddUnhandledExceptionHandler` vs `AddUnhandledExceptionDelegate` as this seemed more fitting.